index.d.ts

```diff
@@ -0,0 +1,92 @@
+import type PostHTML from "posthtml";
+
+interface HtmlnanoOptions {
+  skipConfigLoading?: boolean;
+  collapseAttributeWhitespace?: boolean;
+  collapseBooleanAttributes?: {
+    amphtml?: boolean;
+  };
+  collapseWhitespace?: "conservative" | "all" | "aggressive";
+  custom?: (tree: PostHTML.Node, options?: any) => PostHTML.Node;
+  deduplicateAttributeValues?: boolean;
+  minifyUrls?: boolean;
+  mergeStyles?: boolean;
+  mergeScripts?: boolean;
+  minifyCss?: {
+    preset?: any;
+  };
+  minifyConditionalComments?: boolean;
+  minifyJs?: any;
```

Comment:
Is it possible to import terser type here?

Reply:
Nice catch – [4aadce0](https://github.com/posthtml/htmlnano/pull/203/commits/4aadce0cecf1a52b3d5b9b4b483609d94b56d51d)